libs/mini-rx-store/src/lib/actions.ts

```diff
@@ -26,42 +19,11 @@ export interface ComponentStoreSetStateAction<T> {
     type: string; // The action type visible in DevTools / Logging Extension (really only for logging!)
 }
 
-// Union type
-export type SetStateAction<T> = FeatureStoreSetStateAction<T> | ComponentStoreSetStateAction<T>;
-
-export function createMiniRxActionType(miniRxActionType: MiniRxActionType, featureKey?: string) {
-    return miniRxNameSpace + (featureKey ? '/' + featureKey : '') + '/' + miniRxActionType;
-}
-
 export function createMiniRxAction(
```

Comment:
The creation of Actions in the SignalStore works a little bit differently:

The action is created as a plain object. Example: https://github.com/spierala/mini-rx-store/blob/master/libs/signal-store/src/lib/store-core.ts#L97

createMiniRxActionType is used to create the type. 

It would be nice to keep the creation of Actions identical in MiniRx Store and Signal Store. Then createMiniRxAction can be removed as well.

Reply:
This is really a nice to have change.
It can be done later. Maybe in another PR.

Reply:
Ok, I noted down that this can be tackled later in a separate PR 🙂